<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationDialog</class>
 <widget class="QDialog" name="ConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tracker Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Trackpoint interval</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTimeEdit" name="interval">
       <property name="currentSection">
        <enum>QDateTimeEdit::SecondSection</enum>
       </property>
       <property name="displayFormat">
        <string>mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="changeEnabled">
       <property name="text">
        <string>Above a speed of</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="changeSpeed">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string> km/h</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>use an interval of</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QTimeEdit" name="changedInterval">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::SecondSection</enum>
       </property>
       <property name="displayFormat">
        <string>mm:ss</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>interval</tabstop>
  <tabstop>changeEnabled</tabstop>
  <tabstop>changeSpeed</tabstop>
  <tabstop>changedInterval</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>changeEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>changeSpeed</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>changeEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>changedInterval</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
